ui/app/components/action-card.hbs

```diff
@@ -0,0 +1,101 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
```

Comment:
A representation of a running action instance, or group of peered instances.

Here's 2 action-cards:
![image](https://github.com/hashicorp/nomad/assets/713991/ba29e6b8-e3b6-4c98-8f8c-fe65f4eb5e35)